src/SqlAsyncCollector.cs

```diff
@@ -315,6 +315,12 @@ private static void GenerateDataQueryForMerge(TableInformation table, IEnumerabl
                 else
                 {
                     // ToDo: add check for duplicate primary keys once we find a way to get primary keys.
+                    // JObjects ignore serializer settings (https://web.archive.org/web/20171005181503/http://json.codeplex.com/workitem/23853)
+                    // so we have to manually convert property names to lower case before inserting into the query in that case
```

Comment:
What about case sensitive collations?

Reply:
(even if this works, clarifying that it works in a comment would be helpful to readers)

Reply:
Case sensitive has always been up to the caller to get right - we have no way of knowing what the expected case is. Case insensitive is just always converted to lower case for comparisons for convenience (since the casing still has to be the same, but we can control that for case insensitive)

https://github.com/Azure/azure-functions-sql-extension/blob/main/src/SqlAsyncCollector.cs#L526

Reply:
Thanks @Charles-Gagnon